src/styles/post/PostPreviewModal.scss

```diff
@@ -0,0 +1,3 @@
+.post-preview-modal {
```

Comment:
Will implement some styles later

Reply:
I went through the code and ran the branch locally. Everything looks good! Is it possible to have some global styles for all modals? E.g. title font size?